taxi/test/test_form.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+class TestForm(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="tset",
+            password="test12345",
+        )
+
+        self.client.force_login(self.user)
+
+    def test_search_form_for_car(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test",
+            country="test_country"
+        )
+        Car.objects.create(
+            model="test1",
+            manufacturer=manufacturer
+        )
+        Car.objects.create(
+            model="test2",
+            manufacturer=manufacturer
+        )
+        Car.objects.create(
+            model="test3",
+            manufacturer=manufacturer
+        )
+        cars = Car.objects.all()
+        response = self.client.get(reverse("taxi:car-list") + "?model=test3")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "taxi/car_list.html")
+        self.assertEqual(list(response.context["car_list"]), list(cars))

```

Comment:
u are testing ur search feature, so u a filtered by model to search test3 model, and then u are compare filtered list with list consisting of all cars am i right ? Does this test ur search feature ? 
fix for all tests